lib/game.js

```diff
@@ -0,0 +1,68 @@
+function KeyModel(id){
+  this.id = id;
+}
+
+
+function Key(source, model) {
+  this.model = model;
+  this.source = source;
+  this.el = document.createElement("div");
+  this.el.style.width = "50px";
+  this.el.style.height = "150px";
+  this.el.style.float = "left";
+  this.el.style.border= "solid 1px #5d5e60";
+  this.element = document.createElement("audio");
+  this.element.setAttribute("src", this.source);
+  this.element.setAttribute("type","audio/mpeg");
+  this.el.appendChild(this.element);
+
+//JAK TUTAJ UŻYĆ BIND??
+  document.addEventListener("keydown",(event)=>{
```

Comment:
TO jest okej. Arrow function robi autobind.
ale zawsze mozesz przypisac funkcje i ja zbindowac np tak:
```
this.onKeyDown= function(event) {
  if(event.key == this.model.id){
      this.element.play();
  }
}.bind(this)
document.addEventListener("keydown",this.onKeyDown)
```

albo np.
```
function Key(source, model) {
  /* ...*/
 document.addEventListener("keydown", this.onKeyDow.bind(this) )
}
Key.prototype = {
  render: function(){
    return this.el;
  },
  onKeyDown: function (event) {
    if(event.key == this.model.id){
      this.element.play();
    }
  }
}
```